### ISSUE TITLE
Missing Abilities

### ISSUE THREAD

**Issue by EZtouch** [NightShadeI/Revolution-Rotation-Calculator-#8]
Some of the missing offensive abilities:
Meteor Strike(Ultimate) http://runescape.wikia.com/wiki/Meteor_Strike
Since we can't predict critical strikes (which will restore 10% adrenaline), we should ignore that part of the ability I think 🤔.

Defensive Abilities:- Many people want to include some defensive abilities in their action bar. I think  that they should be defined as the rest of abilities, with their damage set to 0. The algorithm would then find the highest damaging bar with the specified abilities.

**Comment by NightShadeI** [comment]
Yeah there are some abilities that I did not include in the first place - and you are absolutely correct as to why I didn’t include meteor strike - variable adrenaline so can break rotations. As for defensives the more I think about it, although I don’t think optimal rotations will be found with them (and I believe they would normally be activated manually) they still can be used for auto attacks - so I will have a think about this and see if I can improve on it.

**Comment by EZtouch** [comment]
> why I didn’t include meteor strike - variable adrenaline so can break rotations.

Manually activating defensive abilities would alter the adrenaline, thus breaking the cycle too 🤔?

**Comment by NightShadeI** [comment]
They would, which is why I discourage this calculator from being used where moving, about using defensive's are important. I can still include them in the calculator - I just don't see much of a point if it is trying to find optimal rotations and they will not do enough damage to compete. I could provide a message when defensive abilities are entered, I suppose?

**Comment by EZtouch** [comment]
Speaking of abilities that break the cycle, when Pulverise kills an enemy, you get 50% adrenaline back. 🤔😔

**Comment by Robadob** [comment]
These 4 abilities should probably be added, though I doubt they will rank highly.

https://runescape.wiki/w/Scare_Tactics
